Description: Force gradle/utils.gradle to use the local dependencies
 (for some reason gradle-debian-helper fails its injection in this buildscript)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/gradle/utils.gradle
+++ b/gradle/utils.gradle
@@ -32,10 +32,10 @@
 
 buildscript {
     repositories {
-        mavenCentral()
+        maven { url "file:///usr/share/maven-repo" }
     }
     dependencies {
-        classpath "org.ow2.asm:asm:$asmVersion"
+        classpath "org.ow2.asm:asm:debian"
     }
 }
 
